spec/lib/blacklight/search_state_spec.rb

```diff
@@ -402,4 +402,46 @@
       expect(search_state.url_for_document(doc)).to eq doc
     end
   end
+
+  describe '#filters' do
+    context 'pivot facet config but no facet params' do
```

Comment:
What do you think about starting the context with a  "with" or "when" as illustrated by https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording  I find this style to be very readable.

Reply:
Thanks for the review @jcoyne.  I've updated the specs with requested changes.